tests/integration/defs/test_e2e.py

```diff
@@ -1626,6 +1626,9 @@ def parse_output(text):
         while True:
             match = re.search(r"(Generated text: \'(.*?)\')", item,
                               re.MULTILINE)
+            if match is None:
```

Comment:
maybe we can replace with following to capture both types of quotes in one go?

match = re.search(r'(Generated text: ["\'](.*?)["\'])', item, re.MULTILINE)